recipe/magento2.php

```diff
@@ -367,6 +369,47 @@
     add('shared_dirs', get('additional_shared_dirs'));
 });
 
+/**
+ * Update cache ip_prefix on deploy so that you are compiling against a fresh cache
+ * Reference Issue: https://github.com/davidalger/capistrano-magento2/issues/151
+ **/
+desc('Update cache id_prefix');
+task('magento:set_cache_prefix', function () {
+    //get current env config
+    $envConfigString = run("cat {{deploy_path}}/shared/" . ENV_CONFIG_FILE_PATH);
+    $envConfig = eval('?>'.$envConfigString);
+    //set prefix to `alias_releasename_`
+    $prefixUpdate = get('alias') . '_' . get('release_name') . '_';
+
+    //update id_prefix to include release name
+    $envConfig["cache"]["frontend"]["default"]["id_prefix"] = $prefixUpdate;
+    $envConfig["cache"]["frontend"]["page_cache"]["id_prefix"] = $prefixUpdate;
+
+    //Generate configuration array as string
+    $envConfigStr = "<?php return " . var_export($envConfig, true) . ";";
```

Comment:
Please use Symfony's VarDumper https://symfony.com/doc/current/components/var_dumper.html

Reply:
Might need a hand here too. Just couldn't;t work out how to use dump to export the array into a file

Any ideas?